Tactics/Sym.lean

```diff
@@ -152,10 +155,26 @@ def sym1 (curr_state_number : Nat) (_hProgram : Lean.Ident) : Lean.Elab.Tactic.T
           -- (try clear $h_st_pc:ident $h_st_err:ident) --$h_st_program:ident
           -- (intro_change_hyps $h_step_n':ident $program:term $h_st_prefix:str)
           -- (try clear $h_step_n':ident)
-      )))
-
--- sym_n tactic symbolically simulates n instructions from
--- state number i.
+      ))
+    trace[Sym] "Running tactic:\n{stx}"
+    Lean.Elab.Tactic.evalTactic stx
+
+/-- `sym1_i_n i n h_program` will symbolically evaluate a program for `n` steps,
+starting from state `i`, where `h_program` is an assumption of the form:
+`s{i}.program = someConcreteProgam`.
+
+The context is assumed to contain hypotheses
+```
+h_s{i}_err : r StateField.ERR s{i} = .None
+h_s{i}_pc  : r StateField.PC  s{i} = $PC
+h_run      : sf = run $STEPS s0
```

Comment:
@alexkeizer Not an action item right now, but we ought to expect the hypothesis that the stack pointer is aligned too in the near future.